### PR TITLE
gcp - add cloud sql flags resource initial setup

### PR DESCRIPTION
**Pull request by egor-miasnikov**
Added initial classes with `query` and `get` functionality for `Flag` resource in `CloudSQL` service

### CODE REVIEW

**Comment by kapilt** [comment]
Just looked at the docs for these, afaics this appears to be read only metadata on the gcp platform side, what’s the use case?

**Comment by egor-miasnikov** [comment]
@bclements @jondavidi can you help me with use cases?

File: tools/c7n_gcp/c7n_gcp/resources/sql.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2018 Capital One Services, LLC
+# Copyright 2018-2019 Capital One Services, LLC
```
Comment (jondavidi) [inline tools/c7n_gcp/c7n_gcp/resources/sql.py]: this can be just 2019

Comment (egor-miasnikov) [reply tools/c7n_gcp/c7n_gcp/resources/sql.py]: Done

**Comment by jondavidi** [comment]
@egor-miasnikov thank you for the work. Please go ahead and *close* this PR

**Comment by egor-miasnikov** [comment]
Ok thanks!

**Pull request closed without merging**